File: benchmark/Dummy.java
import hlt.*;
import java.util.ArrayList;

/**
 * A dummy bot sends no commands
 */
public class Dummy {

    public static void main(final String[] args) {
        Game game = new Game();
        game.ready("Dummy");

        for (;;) {
            game.updateFrame();
            game.endTurn(new ArrayList<>());
        }
    }
}

File: ml/supervised/dummy.py
import hlt
from hlt import constants
from hlt.positionals import Direction
import random
import logging

# GAME BEGIN

game = hlt.Game()
game.ready("Dummy")

# GAME LOOP

while True:
    game.update_frame()
    command_queue = []
    game.end_turn(command_queue)